src/platform/datapath_raw_socket.c

```diff
@@ -280,8 +280,8 @@ typedef struct ETHERNET_HEADER {
 } ETHERNET_HEADER;
 
 typedef struct IPV4_HEADER {
-    uint8_t VersionAndHeaderLength;
-    uint8_t TypeOfService;
+    UINT8 VersionAndHeaderLength;
```

Comment:
Please use `uint8_t`, not Windows `UINT8` specific.

Reply:
oh right, accidently slipped in from our tcpip code.